examples/VertxExample/README.md

```diff
@@ -0,0 +1,158 @@
+# Tutorial: Getting started with the AWS Advanced Jdbc Driver and Vert.x
+
+In this tutorial, you will set up a Vert.x application with the AWS JDBC Driver, and use the driver to execute some simple database operations on an Aurora PostgreSQL database.
+
+> Note: this tutorial was written using the following technologies:
+>    - AWS JDBC Wrapper 2.2.2
+>    - PostgreSQL 42.5.4
+>    - Java 8
+>    - Vert.x 4.4.2
+
+You will progress through the following sections:
+1. Create a Gradle project
+2. Add the required Gradle dependencies
+3. Set up connection pools
+4. Set up routing
+5. Set up the start method
+6. Run the application
+
+## Step 1: Create a Gradle Project
+Create a Gradle Project with the following project hierarchy:
+```
+├───build.gradle
+└───src
+    └───main
+        └───java
+            └───com
+                └───example
+                    └───starter
+                        ├───model
+                        │   └───Example.java
+                        │   └───Instance.java
+                        └───VertxExampleApplication.java
+```
+
+You can also use the Vert.x initializer to generate a starter project:
+1. Go to https://start.vertx.io/.
+2. Select the Vert.x version, Java, and Gradle.
+3. Add the following dependencies: 
+   - JDBC Client
+   - Vert.x Config
+   - Vert.x Web
+4. Generate the project.
+
+## Step 2: Add the required Gradle dependencies
+Your `build.gradle` file should include the following dependencies:
+
+```gradle
+dependencies {
+    implementation(platform("io.vertx:vertx-stack-depchain:4.4.2"))
+    implementation("io.vertx:vertx-core")
+    implementation("io.vertx:vertx-config")
+    implementation("io.vertx:vertx-jdbc-client")
+    implementation("io.vertx:vertx-web")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.12.7.1")
+    implementation("org.postgresql:postgresql:42.5.4")
+    implementation(project(":aws-advanced-jdbc-wrapper"))
+}
+```
+
+## Step 3: Set up connection pools
+Vert.x applications can use connection pools to handle requests. In this example, we use two connection pools, one to handle write requests and another to handle read requests. Vert.x uses C3P0 by default to manage database connections.
+
+```java
+final JsonObject writeConfig = new JsonObject()
+    .put("url", "jdbc:aws-wrapper:postgresql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:5432/postgres")
+    .put("driverClass", "software.amazon.jdbc.Driver")
+    .put("user", "username")
+    .put("password", "password")
+    .put("minSize", "10")
+    .put("maxSize", "20")
+    .put("testConnectionOnCheckout", "false")
+    .put("testConnectionOnCheckin", "false")
+    .put("idleConnectionTestPeriod", "30")
+    .put("numHelperThreads", "10")
+    .put("max_pool_size", 16);
```

Comment:
```suggestion
    .put("max_pool_size", 20);
```

Reply:
What is the difference between maxSize and max_pool_size?

Reply:
The maxSize and minSize properties were actually from [hibernate specific](https://www.mchange.com/projects/c3p0/#hibernate-specific) properties, they've been removed.